src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -161,6 +164,9 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 				}
 			}
 		}));
+
+		// {{SQL CARBON EDIT}} Extension Recommendation on ADS Launch
+		this.promptADSLaunchRecommendedExtensions();
```

Comment:
Can you check with @alanrenmsft ? I thought we didn't want to enable this yet. I think including this is a mistake so disable for now and just do your own visualizer path instead.